src/TodoList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+let todoList = [
+    { id: 1, title: "Complete assignment 1" },
+    { id: 2, title: "Complete assignment 2" },
+    { id: 3, title: "Complete assignment 3" }
+];
+
+
+let TodoList = function() {
```

Comment:
Please check the syntax for writing an arrow function:

`const TodoList = () => {
  return ( ... )
}`

or write a regular function expression:
`function TodoList() {
  return (
    ...
  )
}`